src/main/java/com/myjeeva/digitalocean/pojo/Volume.java

```diff
@@ -0,0 +1,85 @@
+package com.myjeeva.digitalocean.pojo;
+
+import java.util.Date;
+import java.util.Set;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Volume extends RateLimitBase {
+
+	private static final long serialVersionUID = 3274091633535612517L;
+
+	private String id;
+	
+	private Region region;
+	
+	@SerializedName("droplet_ids")
+	private Set<Integer> dropletIds;
+	
+	private String name;
+	
+	private String description;
+	
+	@SerializedName("size_gigabytes")
+	private Integer sizeGigabytes;
+	
+	@SerializedName("created_at")
+	private Date createdAt;
```

Comment:
Currently [this](https://github.com/jeevatkm/digitalocean-api-java/blob/master/src/main/java/com/myjeeva/digitalocean/pojo/Image.java#L58) name is followed for `created_at` attribute in all POJOs.

It's better to keep consistence.  

_PS: IMHO current variable name is nothing wrong_